### DISCUSSION TITLE
2022-08-10 Weekly Meeting 68 Agenda

### DISCUSSION THREAD

**Discussion by antpb** [omigroup/omigroup#267] [Weekly Meetings]
Weekly Meeting 68 is happening on Wednesday August 10th, 2022 from 9:00 AM PST(16:00 UTC) - 10:00 AM PST (17:00 UTC)

You can add it to your calendar [here](https://calendar.google.com/calendar/u/1?cid=Y18wZHB1Z2Y5ZjgzZXE0cWVrbWI2b21xYmptZ0Bncm91cC5jYWxlbmRhci5nb29nbGUuY29t).
 
We'll be meeting again in the omi-weekly-meeting channel of the [AngellXR/OMI Discord](https://discord.gg/NJtT9grz5E).
We regularly use the # omi-meeting-chat channel for text chat. 

## Who we are and why we're here:

The Open Metaverse Interoperability Group (OMI) is focused on bridging virtual worlds by designing and promoting protocols for identity, social graphs, inventory, and more. Our members include businesses and individuals working towards this common goal. 

If you’re an observer or here for the first time, this is a meeting where people who work on the OMI group initiatives discuss updates and contribution opportunities for current projects and goals. This is your chance to jump in and help!

We welcome you to introduce yourself in #omi-meeting-chat as we dive into the agenda items. There will be an open floor in the last 15 minutes of the meeting (agenda permitting) where we can discuss any items you feel are relevant to the current or near future OMI goals.

## Agenda
- Volunteer to take notes? Volunteer to record? 
- Welcome newcomers
- Check in on MSF

### Working Group Updates:
- [OMI Website update](https://github.com/omigroup/website)
- [Scripting group](https://github.com/omigroup/omi-scripting-group)
  - OMI developer hour
- [glTF Extensions Group](https://github.com/omigroup/gltf-extensions)
  - Progress on OMI_ref and authoring extensions
- [Media Group](https://github.com/omigroup/media)
- any others?

### Discussions: 
- vote in `#omi-vote` channel 
- #222 UX Research Prop 
- select a demo theme for next week!
- recognizing affiliate working groups, and what to do about cancellations and reschedules (ensuring our community is informed).

### Future demo themes: 
- `web-based-FUN` 
- `OMI-audio-emitter` exploration and demo 
- `OMI-collider` exploration and demo 
- `Avatars` Pt II
- `Portals` Pt II 
- `Generative and AI tooling`

**Comment by humbletim** [comment]
mention updated MSF and Scripting Group / Developer Hour meeting times:
- https://github.com/omigroup/omigroup/pull/268
- https://github.com/omigroup/omi-scripting-group/pull/39

**Comment by mrmetaverse** [comment]
![image](https://user-images.githubusercontent.com/63426722/183957710-232ffd78-53a5-46bc-87ef-fe6f5249774d.png)

**Comment by mrmetaverse** [comment]
# Website group 
- gathering content and ideas for ways to add projects or anything else to the website. 
- please visit the [website channel](https://github.com/omigroup/website) and propose your ideas in our [website/discussions](https://github.com/omigroup/website/discussions). 
- Shirley proposed adding MSF affiliation to our website as well. 
- humbletim mentioned that there is this whole universe of editorial review processes in wordpress. Lets explore whats there and what might fit in in conjunction with using github. 
- @antpb regularly shares awesome wordpress updates using omi protocols, maybe we can look into implementing some of those on our site as well.

**Reply by mrmetaverse** [reply]
- maybe sync every page with the respective github readme, so there is a website page synced to their github page.

**Reply by mrmetaverse** [reply]
Created
- to add MSF to our website: 
- to explore wordpress: https://github.com/omigroup/omigroup/issues/274

**Reply by mrmetaverse** [reply]
https://github.com/omigroup/website/issues/57
https://github.com/omigroup/website/issues/56

**Comment by mrmetaverse** [comment]
# Scripting 
- Thursdays 17:00 UTC 
- the calendar is up to date (thanks Tim) 
- proposal for splitting out the developer hour from the scripting group. 
  - Want to ensure it's inclusive, anyone can feel welcome to join, and its not tied to standards creation. 
  - new push on twitter to get more people coming in.  
  - recently Adam shared a bunch of cool stuff they are working on for sinespace during one of our developer hours. A protocol for real-time editing of scenes, replicating scene updates, etc. It sounds very interesting and familiar because we are working on something similar too, if you are working on a networking protocol that has something to do with scene replication, it might be of interest to you. :wink: 
  - they also shared awesome stuff related openid verification and more.

**Comment by mrmetaverse** [comment]
# gLTF 
- [gtltf working group ](https://github.com/omigroup/gltf-extensions)
- continuing to flip through specs we are working on 
  - awaiting more feedback related to the audio_emitter spec. 
  - discussed omi_ref at the last meeting as well. 
- proposed incremental update re: lighting

**Comment by mrmetaverse** [comment]
# Media 
- a few things to look into 
- issues re: bot 
  - producing silent files without audio captured 
  - looking into a backup or alternative
  - will review the output of this session to see if it's ongoing or sporadic  
  - This relates to the [Milestone: Selecting our Bots](https://github.com/omigroup/omigroup/milestone/3)
- have a demo video you'd like help blasting out? Drop it in the `omi-media` channel. 
- tag us `@open_metaverse` and use hashtags `OMI` `openmetaverse` in your tweets and we can help syndicate your announcements as well too.

**Comment by mrmetaverse** [comment]
# MSF-Delegates 
- vote up in the `omi-vote` channel 
- no plenary meeting this week 
- delegates met to discuss our role and shared a proposed list of topics that people in MSF have shared a desire to explore. Which ones are we most interested in supporting? 
![image](https://user-images.githubusercontent.com/63426722/183963376-205d44a0-7b98-4bc7-a81d-2734546b640c.png)
- the MSF plenary meeting is generally 2 hours before the OMI weekly meeting 
- the omi-msf-delegates meeting will be the hour before our OMI weekly meeting every Wednesday, 15:00UTC (see [this PR](https://github.com/omigroup/omigroup/pull/268)). So next week we intend to have a broader discussion, and visual exploration, most likely in Miro/Mural/Figjam (OMI has a Miro account, and has used figjam in the past). 
  - There is a town hall of sorts 
- they mentioned the speaker's bureau in one of their last updates, if you are interested in those opportunities, please let us know in the `msf-delegates` channel.

**Reply by humbletim** [reply]
![image](https://user-images.githubusercontent.com/94753/183964698-c53beed7-d9e8-47b8-9334-a99d8f2ad9b0.png)

**Comment by mrmetaverse** [comment]
# Reviewed: Updated prop for UX Research group 
- #222 
- chance to collab with another group
- we wouldn't be arbitrarily stamping our seal on their work, would need a review process. 
- the research prop from Protocol labs would fall into scope with a UX Research group. 
- Going to give @Elirudite a chance to respond as well. 
- no vote today 
- lets start a thread in the discord. a temp channel. 
- schedule vote for next week to form the working group, the working group can decide on involvement with protocol labs research doc, and what that process looks like. 
- Must confirm critical mass (3-5?), and a lead(s). Point is: are enough people guaranteeing motivation and availability to meet

**Comment by mrmetaverse** [comment]
# housekeeping
## recognizing affiliate working groups, and what to do about cancellations and reschedules (ensuring our community is informed).
- use discord events? 
- we all try to keep our calendars up to date (sometimes balls are dropped, but we do our best) 
- if there is any issue, jump into discord and wave the flag "meeting time is wrong" etc. 
- its still a manual process, but we want to keep meetings consistent. 
- M3 avatar group actually used the OMI template to create a readme and more. Meets every single week, for nearly 4 months. 
- Maybe adding a section to the OMI calendar that points to related calendars for separate subscription too? 
- similar to what humbletim was mentioning before about wordpress, we could build an entire calendar inside of our website that federates all of this. 
- maybe sync every page with the respective github readme, so there is a website page synced to their github page.

**Comment by mrmetaverse** [comment]
# Discord Forums 
- lets test it on a channel

**Comment by mrmetaverse** [comment]
# in person meetup in Oakland in September!

- "maybe we can demo the physical VR machine"

**Comment by mrmetaverse** [comment]
# Next weeks theme: FUN!  
(web based fun, or gaming?) 

Will decide on theme name and announce.

Zesty will demo their new webxr FPS.

**Reply by mrmetaverse** [reply]
[ticket fodder] create a graphic, and make the announcement for next weeks theme (created #272)

**Reply by mrmetaverse** [reply]
As promised 
![image](https://user-images.githubusercontent.com/63426722/184242942-4d04b7fb-1c6b-442d-81e6-1651b2fd96c6.png)
